pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat10_cmd = 'qutech:rotmat{}:rotmat10'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat10'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 10 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat10_cmd + '?',
+                               set_cmd=srotmat10_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat11_cmd = 'qutech:rotmat{}:rotmat11'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat11'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 11 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat11_cmd + '?',
+                               set_cmd=srotmat11_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            '''
+            TV mode parameters
+            '''
+            sintavg_cmd = 'qutech:tvmode{}:naverages'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_naverages'.format(ch_pair),
+                               units='#',
+                               label=('the number of integration averages' +
+                                      'of ch_pair {} '.format(ch_pair)),
+                               get_cmd=sintavg_cmd + '?',
+                               set_cmd=sintavg_cmd + ' {}',
+                               vals=vals.Numbers(1, 1024)
+                               )
+            stvseg_cmd = 'qutech:tvmode{}:nsegments'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_nsegments'.format(ch_pair),
+                               units='#',
+                               label=('the number of TV segments of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stvseg_cmd + '?',
+                               set_cmd=stvseg_cmd + ' {}',
+                               vals=vals.Numbers(1, 256)
+                               )
+            stven_cmd = 'qutech:tvmode{}:enable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_enable'.format(ch_pair),
+                               label=('enable tv mode' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stven_cmd + '?',
+                               set_cmd=stven_cmd
+                               # vals=vals.Numbers(1,4096)
+                               )
+            stvdis_cmd = 'qutech:tvmode{}:disable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_disable'.format(ch_pair),
```

Comment:
The way this parameter is named, it is a function. 

I would say that the corresponding parameter would be 
`ch_pair{}_tvmode_state` (or even better `ch_pair{}_int_avg_enabled`). 

This would then be getable (is the state enabled or disabled?) and setable, you can turn it on and off. 

Right now this is a function instead of a parameter. It should be a parameter.